src/common/cid.ts

```diff
@@ -0,0 +1,26 @@
+import { CID } from "multiformats/cid"
+
+
+/**
+ * Decode a possibly string-encoded CID.
+ * Passing an already decoded CID instance works too.
+ * NOTE: Throws an error if a CID cannot be decoded!
```

Comment:
Nitpick: There's `@throws` you could also use. Not that it really matters that much for the tooling we use - just seems to be the convention.

Reply:
Ah cool, will change.